pkg/cmd/system/events.go

```diff
@@ -81,7 +98,19 @@ func Events(ctx context.Context, client *containerd.Client, options types.System
 				}
 			}
 			if tmpl != nil {
-				out := EventOut{e.Timestamp, e.Namespace, e.Topic, string(out)}
+				var data map[string]interface{}
+				err := json.Unmarshal(out, &data)
+				if err != nil {
+					log.G(ctx).WithError(err).Warn("cannot marshal Any into JSON")
+				}
+
+				id := "unknown"
```

Comment:
should we initialize as "" which is default for golang

Reply:
because i think the clients might expect something like null to verify if the ID exist or not

Reply:
Got it. Changed the default value to be "".